### PR TITLE
Eventify Final Project 

### PR DESCRIPTION
**Pull request by IamYunusAli**
## Eventify
In this PR we have done the following project requirements.
- [Requirementets](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)

### Frontend 

- [Frontend Repo](https://github.com/betses/Final-capstone--frontend)
- [Live Demo](https://eventify-cap.netlify.app/)
- [FrontEnd PR](https://github.com/betses/Final-capstone--frontend/pull/8)

### API 

- [Live API Endpoints Users](https://eventifyhub.herokuapp.com/users)
- [Live API Endpoints Events](https://eventifyhub.herokuapp.com/events)
- [Live API Endpoints Reserves](https://eventifyhub.herokuapp.com/reserves)

-  [API DOCS](https://eventifyhub.herokuapp.com/api-docs/)

### Team
 - @betses
 - @fullstop125 
 - @IamYunusAli 

## kanabnboard
- [Eventfiy](https://github.com/users/IamYunusAli/projects/3)

### Could you follow up on the details in the Readme to set up and run it?
- `seer.rb `is empty at this stage kindly skip that part

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,256 @@
...
+
+Test the models
+```sh
+bundle exec rspec spec/models
+```
+
+Test the API endpoint
+```sh
+rake rswag
+```
```
Comment (Meltrust) [inline README.md:189]: - [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  resources :users
+  resources :events
+  resources :reserves
```
Comment (Meltrust) [inline config/routes.rb]: ## Explanation

By using the `only:` keyword parameter, you can generate only the routes that you are using.

[Look](https://launchschool.com/books/demystifying_rails/read/routes_and_resources):



## Example

```ruby
resources :users, only: [:index, :create, :show]

```

That way, your routes will be more consistent, and only the ones needed will be generated💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
note:  Please go to the frontend PR to view the review comments (working on it).

Hi @IamYunusAli,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  API is working well✔️ 
-  Good readme ✔️ 




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: config/routes.rb (hunk 2 above)
Comment (IamYunusAli) [reply config/routes.rb]: thank you @Meltrust I will manage to update the required changes.

**Review by Meltrust — APPROVED** [review]
#### Hi @IamYunusAli,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Routes have been refactored so only the necessary ones are generated✔️ 
- Test folder used by another framework has been removed✔️ 
- Excellent work, the API works well✔️ 



Cheers and Happy coding!👏👏👏


------

**Pull request merged**